imager/imager_profile/handlers.py

```diff
@@ -0,0 +1,34 @@
+# _*_ coding: utf-8 _*_
+"""signal handlers registered by the imager_users app"""
+
+from __future__ import unicode_literals
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.db.models.signals import pre_delete
+from django.dispatch import receiver
+from imager_profile.models import ImagerProfile
+import logging
+
+logger = logging.getLogger(__name__)
+
+@receiver(post_save, sender=settings.AUTH_USER_MODEL)
+def ensure_imager_profile(sender, **kwargs):
+    # import pdb; pdb.set_trace()
```

Comment:
A minor note about gitflow: commented code should be stripped from your work before you push it up to github. Definitely keep comments in, but chunks of code that are not being used should be removed.